react-ui/src/components/schedule/calendar/Calendar.js

```diff
@@ -8,9 +8,34 @@ import DayView from './DayView';
 import MultipleDaysView from './MultipleDaysView';
 import { addDays } from './dateUtils';
 import { arrayEquals } from '../../../utils/arrays';
+// TODO: set 'NODE_PATH=src/' in .env file to get below to work, (and then refactor a bunch of files):
+// import { arrayEquals } from 'utils/arrays';
+
+// TODO: move these to external consts folder, update CalendarContainer as well.
```

Comment:
Since this is only being used in this file, we can just leave it here.  What do you think?

Reply:
That's true, but I also noticed `schedule/CalendarContainer.jsx` has some constants too that need to be pulled out and are somewhat related.
In my forked repo, in the branch `my-schedule/fix-render-transition` (abandoning these changes for now), I started refactoring it a little and ended up with the following constants defined in `my-schedule/constants/index.js`

```javascript
export const MODE_DAY = 'day';
export const MODE_THREE_DAYS = '3days';
export const MODE_WEEK = 'week';
export const MODE_MOVE = 'move';

 export const DAYS_IN_WEEK = 7;
export const DAYS_IN_THREE_DAYS = 3;

 export const REFERENCE_DATE = new Date(2017, 1, 1);

 export * from './calendarEventTypes'; 
```

and `calendarEventTypes.js` contains:

```javascript
export const USER = 0;
export const FRIEND = 1;
export const SHARED = 2;
```

I'm not sure how many more constants will be used, but IMO it just makes it easier to understand what constants are defined if they're kept separate from their usage even if there aren't that many.